app/controllers/carts_controller.rb

```diff
@@ -1,12 +1,15 @@
 class CartsController < ApplicationController
   def create
     update_cart
-    flash[:notice] = "Added to Cart"
-    redirect_to projects_path
+    flash[:notice] = 'Added to Pending Loans'
+    redirect_to pending_loan_path
```

Comment:
So, in short, Jorge mentioned that "cart" was not a good name for our loans that need to be checked out. So we decided to start changing some of the language we were using for cart to make it more consistent with what we eventually want cart to be named. Instead of cart, we decided to call the cart "pending_loans". Not the best name, but at least it's descriptive.